lib/tools/adb-commands.js

```diff
@@ -707,8 +707,24 @@ methods.getDeviceProperty = async function (property) {
 };
 
 methods.setDeviceProperty = async function (prop, val) {
+  let apiLevel = await this.getApiLevel(), err;
```

Comment:
also, there is a general recommendation to define variables close to the place where they are first used